SpatialGDK/Source/SpatialGDK/Private/Interop/SnapshotManager.cpp

```diff
@@ -89,7 +97,7 @@ FString GetSnapshotPath(const FString& SnapshotName)
 
 // LoadSnapshot will take a snapshot name which should be on disk and attempt to read and spawn all of the entities in that snapshot.
 // This should only be called from the worker which has authority over the GSM.
-void USnapshotManager::LoadSnapshot(const FString& SnapshotName)
+void SpatialSnapshotManager::LoadSnapshot(const FString& SnapshotName)
```

Comment:
Do we also want to rename the files SnapshotManager -> SpatialSnapshotManager?

Reply:
Perhaps after this PR so diffing is still easy?

Reply:
Yes, that makes sense, I'd like to do that in the next PR

Reply:
Added to summary below.